accord-core/src/test/java/accord/coordinate/tracking/TrackerReconcilerTest.java

```diff
@@ -67,7 +67,14 @@ void test(int count, BiFunction<RandomSource, Topologies, ? extends TrackerRecon
     static <ST extends ShardTracker, T extends AbstractTracker<ST>, E extends Enum<E>>
     void test(long seed, BiFunction<RandomSource, Topologies, ? extends TrackerReconciler<ST, T, E>> constructor)
     {
-        for (TrackerReconciler<?, ?, ?> test : TrackerReconciler.generate(seed, constructor))
-            test.test();
+        try
+        {
+            for (TrackerReconciler<?, ?, ?> test : TrackerReconciler.generate(seed, constructor))
+                test.test();
+        }
+        catch (Throwable t)
+        {
+            throw new AssertionError("Failure for seed " + seed, t);
```

Comment:
unrelated to the core work, but was found when doing the topology refactors... the current test relies on logging which means that CI is unlikely to give us the seed we failed with, making theses property tests not reproducible...  by throwing with the seed we should be able to repo issues assuming the property tests are deterministic